src/fsatrace.c

```diff
@@ -122,16 +133,22 @@ main(int argc, char *const argv[])
 		      "   q: dump file stat operations\n"
 		      ,argv[0]);
 	out = argv[2];
-	if ((err = shmInit(&shm, out, LOGSZ, 1)))
+	if ((err = shmInit(&shm, out, buf_size, 1)))
 		fatal("allocating shared memory (%d)", err);
 	snprintf(envout, sizeof(envout), ENVOUT "=%s", out);
 	putenv(envout);
+	snprintf(envbufsize, sizeof(envbufsize), ENVBUFSIZE "=%ld", buf_size);
```

Comment:
This yields warnings on Windows, maybe `(long)buf_size`?

Reply:
fixed